### DISCUSSION TITLE
I wnat to contact with you.

### DISCUSSION THREAD

**Discussion by Hanif131517** [Shashank02051997/QuizzieThunder-Flutter#4] [General]
Hi, good morning, Sir. Sorry for bothering you. I just want to talk to you about your app and make some changes. I hope you will be able to help me to connect Firebase as well. I am waiting for your reply. Thanks, have a good time.

**Comment by Shashank02051997** [comment]
Hi @Hanif131517 
Firstly, thank you for taking the time to check the QuizzeThunder Flutter repo. Sure, I can help you connect with Firebase. Let me know the specific details or changes you’d like to make in the app, and we can get started. Looking forward to assisting you!

**Reply by KazeAZ** [reply]
Good day sir, I just want to ask if is it possible to download or access the firebase version of your app? I hope for your kind response thank you.

**Comment by Hanif131517** [comment]
Hi, thanks you so much for your such a quick response I really appreciate
it. I would like to talk to you about this project. Here is my mobile
number +351920280011.  You can call me in WhatsApp. Or you can add me in
Facebook where I have already sent you a friend request after then you we
can have a chat in messenger. Check out your calendar and book any time you
prefer and let me know please. Thanks again for your kind words, have a
wonderful day.

**Comment by slmnkhn79** [comment]
Hey I wanted to have a connect. Let me know your availability and contact details over email : slmnkhn79@gmail.com
